test/parallel/test-async-wrap-pop-id-during-load.js

```diff
@@ -7,15 +7,17 @@ if (process.argv[2] === 'async') {
     fn();
     throw new Error();
   }
-  (async function() { await fn(); })();
-  // While the above should error, just in case it doesn't the script shouldn't
-  // fork itself indefinitely so return early.
-  return;
+  return (async function() { await fn(); })();
 }
 
 const assert = require('assert');
 const { spawnSync } = require('child_process');
 
-const ret = spawnSync(process.execPath, [__filename, 'async']);
+const ret = spawnSync(
+  process.execPath,
+  ['--stack_size=50', __filename, 'async']
+);
 assert.strictEqual(ret.status, 0);
-assert.ok(!/async.*hook/i.test(ret.stderr.toString('utf8', 0, 1024)));
+const stderr = ret.stderr.toString('utf8', 0, 2048);
```

Comment:
Why the increase in `end`?

Reply:
I checked the actual output and it was cut off relatively early without printing a lot of information. By increasing that limit it actually makes sure we really test for the right values.